apps/web/src/hooks/affine/use-affine-refresh-auth-token.ts

```diff
@@ -38,5 +36,8 @@ export function useAffineRefreshAuthToken(
   useSWR('autoRefreshToken', {
     fetcher: revalidate,
     refreshInterval,
+    revalidateOnFocus: true,
```

Comment:
Is there a minimum revalidate throttle for these flags?

Reply:
https://swr.vercel.app/docs/api#options